spec/policies/company_policy_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe CompanyPolicy do
+  subject { CompanyPolicy.new(user, new_company) }
+
+  let(:new_company) { create(:company) }
```

Comment:
I think here we can use `build_stubbed`, to get a faster test suit

https://thoughtbot.com/blog/use-factory-girls-build-stubbed-for-a-faster-test